adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -2377,12 +2657,61 @@ abstract class MultisampleAvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Prod
 }
 
 /**
- * An abstract class that extends GenomicRDD and where the underlying data are
+ * An abstract class that extends GenomicDataset and where the underlying data are
  * Avro IndexedRecords. This abstract class provides methods for saving to
  * Parquet, and provides hooks for writing the metadata.
  */
-abstract class AvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Product, V <: AvroGenomicRDD[T, U, V]] extends ADAMRDDFunctions[T]
-    with GenomicDataset[T, U, V] {
+abstract class AvroGenomicDataset[T <% IndexedRecord: Manifest, U <: Product, V <: AvroGenomicDataset[T, U, V]] extends GenomicDataset[T, U, V] {
+
+  protected def saveRddAsParquet(args: SaveArgs): Unit = {
+    saveRddAsParquet(
+      args.outputPath,
+      args.blockSize,
+      args.pageSize,
+      args.compressionCodec,
+      args.disableDictionaryEncoding
+    )
+  }
+
+  /**
+   * Saves an RDD of Avro data to Parquet.
+   *
+   * @param pathName The path to save the file to.
+   * @param blockSize The size in bytes of blocks to write. Defaults to 128 * 1024 * 1024.
+   * @param pageSize The size in bytes of pages to write. Defaults to 1 * 1024 * 1024.
+   * @param compressCodec The compression codec to apply to pages. Defaults to CompressionCodecName.GZIP.
+   * @param disableDictionaryEncoding If false, dictionary encoding is used. If
+   *   true, delta encoding is used. Defaults to false.
+   * @param optSchema The optional schema to set. Defaults to None.
+   */
+  protected def saveRddAsParquet(
+    pathName: String,
+    blockSize: Int = 128 * 1024 * 1024,
+    pageSize: Int = 1 * 1024 * 1024,
+    compressCodec: CompressionCodecName = CompressionCodecName.GZIP,
+    disableDictionaryEncoding: Boolean = false,
+    optSchema: Option[Schema] = None): Unit = SaveAsADAM.time {
```

Comment:
would be nice to have separate SaveRddAsParquet and SaveDatasetAsParquet timers

Reply:
We're not really going to get meaningful timing info out of the `SaveDatasetAsParquet` timer.

Reply:
ok